src/GeneralStateTestsFiller/stCreateTest/CREATE_EOF1DeployValidPUSHOpcodesFiller.yml

```diff
@@ -0,0 +1,522 @@
+CREATE_EOF1DeployValidPUSHOpcodes:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 200000000
+      code: ''
+      nonce: 0
+      storage: {}
+    b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      code: |
+        :yul 
+        {
+          let init_code_len := calldatasize()
+          calldatacopy(0, 0, init_code_len)
+          if iszero(create(0, 0, init_code_len)) { sstore(1, 1) }
```

Comment:
It's better to `sstore(1, 1)` if create was successful, and check that.

Reply:
Changed.